notebook/static/notebook/js/actions.js

```diff
@@ -90,139 +94,164 @@ define(function(require){
-            help: 'restart the kernel and clear all output (with dialog)',
+        	cmd: i18n._('confirm restart kernel and clear output'),
+            help: i18n._('restart the kernel and clear all output (with dialog)'),
             handler: function (env) {
                 env.notebook.restart_clear_output();
             }
         },
         'interrupt-kernel':{
             icon: 'fa-stop',
+            cmd: i18n._('interrupt the kernel'),
+            help: i18n._('interrupt the kernel'),
             help_index : 'ha',
             handler : function (env) {
                 env.notebook.kernel.interrupt();
             }
         },
         'run-cell-and-select-next': {
+            cmd: i18n._('run cell and select next'),
             icon: 'fa-step-forward',
-            help    : 'run cell, select below',
+            help: i18n._('run cell, select below'),
             help_index : 'ba',
             handler : function (env) {
                 env.notebook.execute_cell_and_select_below();
             }
         },
         'run-cell':{
-            help    : 'run selected cells',
+            cmd: i18n._('run selected cells'),
+            help    : i18n._('run selected cells'),
             help_index : 'bb',
             handler : function (env) {
                 env.notebook.execute_selected_cells();
             }
         },
         'run-cell-and-insert-below':{
-            help    : 'run cell, insert below',
+            cmd: i18n._('run cell and insert below'),
+            help    : i18n._('run cell and insert below'),
             help_index : 'bc',
             handler : function (env) {
                 env.notebook.execute_cell_and_insert_below();
             }
         },
         'run-all-cells': {
-            help: 'run all cells',
+            cmd: i18n._('run all cells'),
+            help: i18n._('run all cells'),
             help_index: 'bd',
             handler: function (env) {
                 env.notebook.execute_all_cells();
             }
         },
         'run-all-cells-above':{
+            cmd: i18n._('run all cells above'),
+            help: i18n._('run all cells above'),
             handler : function (env) {
                 env.notebook.execute_cells_above();
             }
         },
         'run-all-cells-below':{
+            cmd: i18n._('run all cells below'),
+            help: i18n._('run all cells below'),
             handler : function (env) {
                 env.notebook.execute_cells_below();
             }
         },
         'enter-command-mode': {
-            help    : 'command mode',
+            cmd: i18n._('enter command mode'),
+            help    : i18n._('enter command mode'),
             help_index : 'aa',
             handler : function (env) {
                 env.notebook.command_mode();
             }
         },
         'insert-image': {
-            help      : 'insert image',
+            cmd: i18n._('insert image'),
```

Comment:
No, it's actually a necessary addition. There are some cases where the command name tries to show up ( I think on the keyboard shortcuts dialog ), but was not translatable because the displayed name was just being synthesized from the internal name ( i.e. "insert-image" becomes "insert image" ).  And in some cases, you need both a short command name, and a longer help message, which is why I have 2 separate fields for them now.